src/shared/components/palette/base-field/field-read.component.ts

```diff
@@ -33,6 +41,11 @@ export class FieldReadComponent extends AbstractFieldReadComponent implements On
 
     // Provide component @Inputs
     component.instance['caseField'] = this.caseField;
+    // console.log('this.caseField', this.caseField);
```

Comment:
Can we remove these commented console.logs?

Reply:
Will remove all of them.